### DISCUSSION TITLE
Bounding box longitudes are more than 1 degree apart

### DISCUSSION THREAD

**Discussion by deepu9** [Leaflet/Leaflet#9082] [Q&A]
Hi,
Thanks for all your work with leafletjs. With leafletjs, I don't need to worry about building maps. All I've to do is give a url, viola, I get a map. Also, it's quite easy to access the map data using objects.

I'm trying to build a bounding box for zoom levels above 9. When I call bounding box methods, I'm getting lat and lng for all four corners, but the longitude difference is more than 1 degree apart. The third party maps is throwing the following error:

**maximum width and height of a bounding box can be 1 degree**

```javascript
import React, { useRef } from "react";

const map = useRef({});
const currentMap = map.current;
const bounds = currentMap.leafletElement.getBounds();

bounds.toBBoxString() -> "143.0388971008785,-38.446571163086084,146.768755499316,-37.19038041243975"
```

**leafletjs version: 1.6.0**

Can you help me out. Thanks

**Comment by IvanSanchez** [comment]
> The third party maps is throwing the following error: `maximum width and height of a bounding box can be 1 degree`

Ask this mysterious third-party map provider how to proceed in this case.

**Reply by deepu9** [reply]
I'm using here maps. When the longitude or latitude difference is more than 1 degree, here maps throws the above error. If I change the longitude and/or latitude, so that difference between two longitudes and two latitudes is less than or equal to 1 degree, then the here maps will send the data back. Is there a way to fix the difference in Leafletjs?

Thanks
